package.json

```diff
@@ -113,6 +113,14 @@
                 }
             }
         },
+        "configurationDefaults": {
+            "[motoko]": {
+                "editor.colorDecorators": false
+            },
+            "[candid]": {
```

Comment:
I don't think this is necessary. Candid uses `variant Bee` as that value and there is no `#` character to write.

Reply:
I went back and forth on it and decided to just include it in case the color decorators show up in strings, comments, etc.